README.md

```diff
@@ -110,6 +110,21 @@ When downloading or uploading for the first time, the welcome page will automati
 
 Once you select upload, after uploading the settings. You will see the Summary details with the list of each files and extensions uploaded.
 
+If you are using Settings Sync on multiple clients, it is possible for you to upload settings to your gist from one client that are

```

Comment:
It would be great if you can remove it from `Readme` and put in Public GitHub Wiki where documentation is required.

Reply:
I can do that later today or tomorrow.  I wasn't aware of the conventions.  This info is sort of specific to this release.  Is it OK if it appears in the wiki before it is released?   Or, do I need to add "As of v3.4.3..."?

Reply:
No problem, yes you can add "As of v3.4.3" for now. We will remove it when it becomes standard after being released

Reply:
OK, done.  Content added to gist and I removed the update to the README from the commit and force-pushed.